### PR TITLE
Add UserDetails and UserCredentials components

### PR DESCRIPTION
**Pull request by cipick**

### CODE REVIEW

File: store/reducers/signup/index.ts [hunk 1]
```diff
@@ -20,6 +20,20 @@ export const userTypeOptions = [
   UserType.ngo,
 ]
 
+export const userTypeForms = {
+  [UserType.individual]: [
+    { name: 'name', label: 'Nume' },
```
Comment (cipick) [inline store/reducers/signup/index.ts:25]: will need to use a translationKey instad of label

File: components/SignUpServicesForm/SignUpServicesForm.tsx [hunk 2]
```diff
@@ -24,6 +24,8 @@ type ServicesForm = {
 
 const SignUpServicesForm = () => {
   const { t } = useTranslation()
+  const { data } = useServicesForm();
+  console.log('useServicesForm data', data);
```
Comment (bogdanmosica) [inline components/SignUpServicesForm/SignUpServicesForm.tsx:28]: Remove console

File: components/SignupVolunteering/index.tsx [hunk 3]
```diff
@@ -4,9 +4,12 @@ import { useTranslation } from 'react-i18next'
 import Checkbox from '@/components/Form/Checkbox'
 import CheckboxWithDescription from '@/components/SignupVolunteering/CheckboxWithDescription'
 import Textarea from '@/components/Form/Textarea'
+import { useVolunteeringForm } from '@/hooks/useData'
 
 const SignupVolunteering: FC = () => {
   const { t } = useTranslation()
+  const { data } = useVolunteeringForm()
+  console.log('useVolunteeringForm data', data)
```
Comment (bogdanmosica) [inline components/SignupVolunteering/index.tsx:12]: same here

**Review by rhadu — APPROVED** [review]
Looking good!

File: components/UserCredentials/UserCredentials.tsx [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+            errors={
+              serverErrors[input.name]
+                ? { message: serverErrors[input.name].join('\n') }
+                : errors[input.name]
+            }
+            type={input.type}
+            {...register(input.name)}
+          />
+        ))}
+        {/*{serverErrors.map((error) => (*/}
```
Comment (bogdanmosica) [inline components/UserCredentials/UserCredentials.tsx:123]: Remove commented code if we don't need

**Pull request merged**